jaxrs/src/main/resources/schema/jboss-as-jaxrs_3_1.xsd

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="urn:jboss:domain:jaxrs:3.1"
```

Comment:
Unless @jamezp disagrees, please do a major version bump for this and the management API version.

That's our standard practice; we don't try and signal the significance of a new version by using major or minor versions.

Leave minor versions to downstream in case they need to insert a new version during their long period between feature releases.